src/webui/www/public/index.html

```diff
@@ -5,11 +5,11 @@
     <meta charset="UTF-8" />
     <title>qBittorrent QBT_TR(Web UI)QBT_TR[CONTEXT=OptionsDialog]</title>
     <link rel="icon" type="image/png" href="images/skin/qbittorrent32.png" />
-    <link rel="stylesheet" type="text/css" href="css/login.css?v=${VERSION}" />
+    <link rel="stylesheet" type="text/css" href="css/login.css?v=1" />
     <noscript>
-        <link rel="stylesheet" type="text/css" href="css/noscript.css?v=${VERSION}" />
```

Comment:
I like the privacy win here, but manually specifying the version number could be troublesome. I think a unique code generated on app startup provides a better balance between cache length and effectiveness.

Reply:
Glad you mention it, I've thought about that before but wasn't sure we would all agree, now is finally the chance.
PR updated.